reanalysis/query_panelapp.py

```diff
@@ -189,6 +196,84 @@ def read_panels_from_participant_file(panel_json: str) -> set[int]:
     return panel_set
 
 
+def find_core_panel_version() -> str | None:
+    """
+    take the default panel ID from config
+    iterate through its associated activities
+    return the panel version which was closest to 12 months ago
+
+    or return None, i.e. if the panel is not >= 12 months old
+
+    Returns:
+        a version string from 12 months prior
+    """
+
+    date_threshold = datetime.today() - relativedelta(years=1)
```

Comment:
Can we (now or in the future) make this configurable to be either X months old or from a specific date?  I am not worried if you add the logic now or merge as is then include this down the track.

Reply:
Doable! I'll stick something in the conf to have 'months' - `relativedelta` works fine with any number of months, not capped at 12